src/pudl/transform/eiaaeo.py

```diff
@@ -178,6 +241,41 @@ def _decumulate(df: pd.DataFrame, decumulate_columns: list[str]) -> pd.DataFrame
             columns=[f"cumulative_{c}" for c in decumulate_columns]
         )
 
+    def _check_totals_add_up(unstacked) -> None:
```

Comment:
I pulled this function into the asset scope since the thresholds + specific columns are all very asset-specific.